### PR TITLE
Add release notes for edge-19.2.1

### PR DESCRIPTION
**Pull request by kleimkuhler**
Dependencies: None

Signed-off-by: Kevin Leimkuhler <kevin@kleimkuhler.com>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
+## edge-19.2.1
+
+* Controller
+  * **Breaking change** Changed automatic proxy injection to be opt-in via the
+    `linkerd.io/inject` annotation on the pod or namespace. More info:
+    https://linkerd.io/2/proxy-injection/
+  * Added `linkerd.io/created-by` annotation to the linkerd-cni DaemonSet
```
Comment (siggy) [inline CHANGES.md]: add `(thanks @codeman9!)`

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+  * Added a 10 second keepalive default to resolve dropped connections in Azure
+    environments
+  * Improved node selection for installing the linkerd-cni DaemonSet
```
Comment (siggy) [inline CHANGES.md]: add `(thanks @codeman9!)`

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+  * `ServiceProfile`s defined in the client namespace now take priority over
+    ones defined in the server namespace
+  * Corrected the expected controller identity when configuring pods with TLS
+  * Modified klog to be verbose when controller log-level is set to `Debug`
+* CLI
+  * Added the `linkerd endpoint` command to introspect Linkerd's service
```
Comment (siggy) [inline CHANGES.md]: `linkerd endpoints`

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+    discovery state
+  * Added the `--tap` flag to `linkerd profile` to generate a service profile by
```
Comment (siggy) [inline CHANGES.md]: replace `service profile` with `ServiceProfile` for consistency.

File: CHANGES.md [hunk 5]
```diff
@@ -1,3 +1,41 @@
...
+    to disable injection for specific pods when running `linkerd inject`
+  * Added support for `basePath` in OpenAPI 2.0 files when running `linkerd
+    profile --open-api`
+  * Increased `linkerd check` client timeout from 5 seconds to 30 seconds to fix
+    issues for clusters with a slower API server
+  * `linkerd routes` will no longer return rows for `ExternalName` services in
+    the namespace
+  * Broadened set of valid URLs when connecting to the Kubernetes API
+  * Improved `ServiceProfile` field validation in `linkerd check`
+* Proxy
```
Comment (hawkw) [inline CHANGES.md]: * Fixed a bug where proxies could output duplicate metrics.
* Added `l5d-client-id` and `l5d-server-id` headers

Comment (klingerf) [reply CHANGES.md]: What do the `l5d-client-id` and `l5d-server-id` headers actually do? Can we provide more info about how users would actually interact with them? Or are they in support of ongoing proxy TLS work?

File: CHANGES.md [hunk 6]
```diff
@@ -1,3 +1,41 @@
+## edge-19.2.1
+
```
Comment (wmorgan) [inline CHANGES.md]: In previous edge releases with breaking changes, we've included a "Upgrade notes" block here. Is that relevant here? E.g. if someone is using proxy injection already, do they have to do something special to upgrade?

Comment (klingerf) [reply CHANGES.md]: The first bullet point with the breaking change includes a link to https://linkerd.io/2/proxy-injection/, which explains the new setup. I think that's probably good enough.

Comment (wmorgan) [reply CHANGES.md]: Works for me

**Pull request merged**